<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facturacion</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FACTURACIÃ“N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_agregar_factura">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AGREGAR</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modificar_factura">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MODIFICAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eliminar_factura">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ELIMINAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar_factura">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>911</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>NUEVA FACTURA</string>
   </property>
   <widget class="QCheckBox" name="fPago_Marzo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MARZO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fMonto">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Octubre">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OCTUBRE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_apenom_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CLIENTE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_telefono">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DNI:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Julio">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JULIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fDNI">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Agosto">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AGOSTO</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Septiembre">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SEPTIEMBRE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Noviembre">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NOVIEMBRE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Diciembre">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DICIEMBRE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fServicio">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_apenom_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PAGO DEL MES DE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fnomApe">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Junio">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JUNIO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_dni_factura">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>SALIDA</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="fEgreso">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>EGRESO</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="fDescripcion">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Abril">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ABRIL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_serv_factura">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_apenom_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DESCRIPCIÃ“N:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxIngreso">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>ENTRADA</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="fIngreso">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>INGRESO</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="fNumFact">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_apenom_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>METODO DE PAGO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_apenom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SERVICIO:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Mayo">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MAYO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_apenom_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FECHA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_apenom_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MONTO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dni">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NÂ° DE FACTURA:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Enero">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ENERO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fMetodoPago">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EFECTIVO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TARJETA DE DEBITO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TARJETA DE CREDITO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHEQUE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DEBITO AUTOMATICO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OTRO</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="fFecha">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="fPago_Febrero">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FEBRERO</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>911</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>FACTURAS DE HOY</string>
   </property>
   <widget class="QTableWidget" name="fTabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>891</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
